geode-core/src/main/java/org/apache/geode/internal/tcp/ConnectionTable.java

```diff
@@ -400,6 +401,10 @@ private Connection getSharedConnection(InternalDistributedMember id, boolean sch
           throw new IOException("Cannot form connection to alert listener " + id);
         }
 
+        if (onlyOneTry) {
```

Comment:
This doesn't feel like an appropriately named variable here. It feels more like a "do not wait for connection".